scripts/init_db.sql

```diff
@@ -0,0 +1,96 @@
+CREATE TABLE books (
+    id SERIAL PRIMARY KEY,
+    isbn TEXT UNIQUE,
+    title TEXT NOT NULL,
+    synopsis TEXT NOT NULL,
+    publisher TEXT NOT NULL,
+    publishing_date TEXT NOT NULL,
+    edition TEXT NOT NULL
+);
+
+CREATE TABLE authors (
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    birth_date TEXT NOT NULL
+);
+
+CREATE TABLE genres (
+    id SERIAL PRIMARY KEY,
+    genre TEXT NOT NULL
+);
+
+CREATE TABLE authored (
+    id SERIAL PRIMARY KEY,
+    book_id INTEGER REFERENCES books (id),
+    author_id INTEGER REFERENCES authors (id)
+);
+
+CREATE TABLE book_genres (
+    id SERIAL PRIMARY KEY,
+    book_id INTEGER REFERENCES books (id),
+    genre_id INTEGER REFERENCES genres (id)
+);
+
+CREATE TABLE libraries (
+    id SERIAL PRIMARY KEY,
+    owner TEXT NOT NULL,
+    name TEXT NOT NULL
+);
+
+CREATE TYPE ReadingStatus AS ENUM ('PLAN_TO_READ', 'CURRENTLY_READING', 'COMPLETED', 'DROPPED', 'ON_HOLD', '');
+
+CREATE TABLE library_entries (
+    id SERIAL PRIMARY KEY,
+    library_id INTEGER REFERENCES libraries (id) ON DELETE CASCADE,
+    book_id INTEGER REFERENCES books (id),
+    score INTEGER,
+    reading_status ReadingStatus NOT NULL DEFAULT ''
+);
+
+CREATE TABLE user_recommendations (
+    id SERIAL PRIMARY KEY,
+    book1_id INTEGER REFERENCES books (id),
+    book2_id INTEGER REFERENCES books (id),
+    CHECK (book1_id <> book2_id)
+);
+
+CREATE TABLE user_recommendation_comments (
+    id SERIAL PRIMARY KEY,
+    recommendation_id INTEGER REFERENCES user_recommendations (id),
+    author TEXT NOT NULL,
+    comment TEXT NOT NULL,
+    score INTEGER NOT NULL DEFAULT 0
```

Comment:
Realmente esto creo que es complicar excesivamente un PMV. ¿Por qué una puntuación y no simplemente un "like"? ¿Y para qué el comentario?

Reply:
La puntuación se usa para valorar el peso de las recomendaciones en el sistema de recomendación automática, mientras que la justificación es información útil para los usuarios que busquen estas recomendaciones, y también se puede usar para ver si una recomendación es troll y moderarla ya sea borrándola o a saber.

Las historias de usuario para todos estos temas ya deberían estar de antes, he visto por encima los issues que has creado (estoy en el curro) y juraría que todo este tema ya se justificó y explicó cuando se metió el data manager que en vez de BD tenía unos json guardados a pelo

Reply:
> La puntuación se usa para valorar el peso de las recomendaciones en el sistema de recomendación automática, mientras que la justificación es información útil para los usuarios que busquen estas recomendaciones, y también se puede usar para ver si una recomendación es troll y moderarla ya sea borrándola o a saber.
> 
> Las historias de usuario para todos estos temas ya deberían estar de antes, he visto por encima los issues que has creado (estoy en el curro) y juraría que todo este tema ya se justificó y explicó cuando se metió el data manager que en vez de BD tenía unos json guardados a pelo

Vamos a ver, las historias de usuario sirven no sólo para satisfacer al usuario, sino también para gestionar el conocimiento correctamente.  En general, si no sabes por qué has añadido algo o no lo recuerdas, es difícil justificar que eso satisface al usuario. Pero lo principal es que es difícil para entender, el que lo revisa, si es correcto o no desde el punto de vista formal o si avanza hacia el producto corrrespondiente. En este PR es complicado, primero, porque no está añadido a un milestone. Pero el problema es que este código específico hace referncia a #87, que *no* está ligado a ninguna historia de usuario, y  no formula un problema. Ese issue hace referencia a #86, que una vez más, no hace referencia a ningún problema real, pero lo más importante, no está ligado a ninguna historia de usuario.
Buscar [recommendation weights](https://github.com/search?q=repo%3AAnglepi%2FMy-Many-Reads%20recommendation%20weights&type=code), "recommendation" only one issue, "weights" a commit message that talks about "avoiding decimal weights". But still is referred to 2f09c3a2127d19d2eaeab7b5dfe81eeb9e86ced9, that *keeps* using float as a representation for the weight anyway. And it refers to #67, that refers to #66, that, again *does not refer to any user story*, which makes impossible, at least for me, to know if what you're doing contributes to solving a user problem or not.

Y no sé por qué diablos he cambiado a inglés, supongo que por la frase del commit. En todo caso, volvemos a lo de antes. Está claro que tú tienes en tu cabeza lo que quieres hacer, y tienes suficiente experiencia para sacarlo adelante. Pero para mi es imposible aportar algún valor si lo que quieres hacer no está alineado con las historias de usuario, y no sigue un proceso adecuado; entramos en un juego de "esto por qué es", "es para no sé qué", "no sé por qué no sé qué ayuda al usuario", "sí, ayuda por no se qué otra cosa y" y por el camino, no resulta una inversión muy rentable ni de tu tiempo ni del mío.